crates/bevy_window/src/window.rs

```diff
@@ -5,6 +5,26 @@ use raw_window_handle::RawWindowHandle;
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub struct WindowId(Uuid);
 
+/// Behavior of the presentation engine based on frame rate.
+#[repr(C)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum PresentMode {
+    /// The presentation engine does **not** wait for a vertical blanking period and
+    /// the request is presented immediately. This is a low-latency presentation mode,
+    /// but visible tearing may be observed. Will fallback to `Fifo` if unavailable on the
+    /// selected  platform and backend. Not optimal for mobile.
+    Immediate = 0,
```

Comment:
I'm not sure I love labelling these enum variants with numbers. What benefit does this serve?

Reply:
To remain binary compatible with the `wgpu` struct. 

https://docs.rs/wgpu-types/0.12.0/src/wgpu_types/lib.rs.html#2632-2647

Reply:
Makes sense. Could you please leave a comment in the code explaining this so it doesn't get carelessly reverted?

Reply:
Oops, I missed this earlier. I've added a note here and pushed a new commit.